vocode/streaming/streaming_conversation.py

```diff
@@ -1465,7 +1472,9 @@ async def send_speech_to_output(
         # Added stop event check otherwise it will block other synthesis result tasks
         # even though we meant to cancel this one.
         sleep_interval = 0.1  # Mark last action every 0.1 seconds
-        remaining_sleep = total_time_sent
+        remaining_sleep = (
+            total_time_sent  # - self.get_duration_spoken_seconds(time_started_speaking)
+        )
```

Comment:
I am now fine with doing this. This would be the only change in this pr thats worth heavy regression testing.

Reply:
aighty